### ISSUE TITLE
Compatibility with Rails 5

### ISSUE THREAD

**Issue by januszm** [rails/rails-perftest#39]
`sub_test_task.rb` was removed in Rails 5.0

```
$ rake
LoadError: cannot load such file -- rails/test_unit/sub_test_task
rails-perftest-0.0.6/lib/rails/perftest/railties/testing.tasks:1:in `<top (required)>'
```

See: https://github.com/Mehonoshin/draper/pull/2/files

For possible solution. Until this is resolved the following monkey patch can be used:

``` ruby
+++ lib/rails/test_unit/sub_test_task.rb

# Monkey patch, gem rails-perftest
require 'rake/testtask'
class Rails::SubTestTask < Rake::TestTask
end
```

**Comment by codered** [comment]
Just for future reference to any readers, just create a directory within your rails "lib" folder (lib/rails/test_unit/sub_test_task.rb). Put those 3 lines mentioned above and you will get past the error. 

This is if your project is using rails 5.

**Comment by senny** [comment]
Should be addressed by 8f8d84489b79ece2afac3f7ddcda2e08c80b9346

The fix was released with 0.0.7
